bigchaindb/common/transaction.py

```diff
@@ -649,54 +707,27 @@ def create(cls, owners_before, owners_after, metadata=None, asset=None,
             raise TypeError('`owners_before` must be a list instance')
         if not isinstance(owners_after, list):
             raise TypeError('`owners_after` must be a list instance')
+        if len(owners_before) == 0:
```

Comment:
That issue was closed. There is no use case where owners_before can be an empty list